### DISCUSSION TITLE
Email service

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cloud#52] [Feature Requests & Enhancements]
It would be nice to provide some sort of first-party email-based solution to Cloud projects (even if just for things like password resets, account verification, etc.) so that you don’t have to set up a 3rd party email provider.

**Comment by AhoyLemon** [comment]
Agreed with @angrybrad on this one. Setting up a new hosting instance and getting a big ugly error message when I try to set up my first account wasn't a great onboarding experience.
